src/action.js

```diff
@@ -0,0 +1,182 @@
+import * as os from "os";
+
+import * as github from "@actions/github";
+import * as toolCache from "@actions/tool-cache";
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+
+// Define consts
+
+export const gruntworkOrg = "gruntwork-io";
```

Comment:
Nit: In our other codebases when we're defining static global variables we have been naming them using all-caps, camel case. E.g: `gruntworkOrg` -> `GRUNTWORK_GITHUB_ORG`

Reply:
Ah good point - fixed